### ISSUE TITLE
Linting not working as expected

### ISSUE THREAD

**Issue by martov1** [alm-tools/alm#217]
Linting does not seem to work out of the box, i tried installing tslint globally and provinding a tslint.json file but i get no warnings

![004b89f2-3ad4-11e6-84dd-82ec1fe53e2b](https://cloud.githubusercontent.com/assets/7854453/19005407/ddd57d3e-8730-11e6-8066-0055b96f5e92.jpg)

**Comment by martov1** [comment]
it seems it does not work when using js support #216

**Comment by basarat** [comment]
What are the contents of `tslint.json` :rose:

**Comment by martov1** [comment]
```
{
  "extends": "tslint:latest",
  "rules": {
    "no-console":"allow",
    "interface-name": false,
    "max-line-length": [true, 140],
    "member-ordering": [true,
      "public-before-private",
      "static-before-instance",
      "variables-before-functions"
    ]
  }
}
```

**Comment by basarat** [comment]
> i tried installing tslint globally

Don't need to do that. Alm ships with the latest tslint. 

I gave it a quick go : 

![image](https://cloud.githubusercontent.com/assets/874898/19009072/9804bc6a-87b3-11e6-995b-d53367627616.png)


But it worked as expected. There is probably a bug here though. The tslint support has been among the lowest priority for me but that is just because no one else seems to be using it, TypeScript is already a pretty powerful linter :). 

Would like a sample project with a reproduction please. 

# My sample: 

`tslint.json`
```
{
  "extends": "tslint:latest",
  "rules": {
    "no-console":"allow",
    "interface-name": false,
    "max-line-length": [true, 140],
    "member-ordering": [true,
      "public-before-private",
      "static-before-instance",
      "variables-before-functions"
    ]
  }
}
```
`tsconfig.json`
```
{
  "compilerOptions": {
    "allowJs": true,
    "outDir": "./dist"
  }
}
```
`./src/foo.ts`
```
export const foo = 123;
if (true) { };
```
`./src/foo.js`
```
export const foo = 123;
if (true) { };
```

**Comment by martov1** [comment]
here is a test project that displays the bug.
[src.zip](https://github.com/alm-tools/alm/files/505452/src.zip)

**Comment by basarat** [comment]
A file in the `outDir` is making its way into the compilation context. `outDir` should always be excluded. Released with `2.2.1` :rose:

Thanks for the great repro :heart:
